scripts/helper-image.sh

```diff
@@ -3,11 +3,14 @@
 set -eou pipefail
 
 help() {
-  echo "Usage: $0 --hostname [hostname] --identity [1|2|3]"
+  echo "Usage: $0 --hostname [hostname] --identity [1|2|3] --platform [platform]"
   echo "- hostname: public hostname that will appear on TLS certificate"
   echo "- identity: helper identity to be fixed inside the IPA build"
+  echo "- platform: the platform for the helper image, defaults to linux/amd64"
 }
 
+platform="linux/amd64"
```

Comment:
We dont need this line right?

Reply:
mm, can you elaborate more? I am not sure I understand the source for confusion here